src/platform/intel/cavs/lps_wait.c

```diff
@@ -17,6 +17,15 @@
 #include <config.h>
 #include <stdint.h>
 
+#if CONFIG_LPS_COUNTERS
+struct lps_dbg {
+	uint32_t task_cnt;
+	uint32_t int_cnt;
+};
+
+struct lps_dbg *_dbg = (struct lps_dbg *)(0x9e004008);
```

Comment:
magic number

Reply:
An offset should be defined and used with mailbox_sw_reg_write() macro.